src/FileUpload.js

```diff
@@ -0,0 +1,115 @@
+
+import React, {useState} from "react"
+
+// Import from "@inrupt/solid-client"
+import {
+  saveFileInContainer,
+  getSourceUrl
+} from '@inrupt/solid-client';
+
+// Import from "@inrupt/solid-client-authn-browser"
+import {
+  fetch
+} from '@inrupt/solid-client-authn-browser';
+
+
+function FileUpload(props)
+{
+	let currentPath = props.currentPath;
+	let [selectedFiles, setSelectedFiles] = useState([]);
+
+
+	function upload() 
+	{
+	    if(selectedFiles.length != 0)
+	    {
+	        console.log("selected file(s):");
+	        for (let file of selectedFiles)
+	        {
+	            console.log(file.name);
+	        }
+	    }
+
+	    console.log("uploading files ...");
+	    for (let file of selectedFiles)
+	    {
+	      	placeFileInContainer(file, currentPath);
+	    }
+	}
+
+
+	/**
+	 * Upload file into the targetContainer.
```

Comment:
I'm not against comments but not sure this one helps

Reply:
it's a reflex :) I just add default javadoc/doxygen comments to the functions generally. But yeah in this case this doesn't help much, we will probably have to improve it